src/app.css

```diff
@@ -42,8 +42,13 @@ body {
   overscroll-behavior: none;
 }
 
-button {
+input {
```

Comment:
We should probably have `input` CSS scoped to the modal. The `input` in `WorkspaceHeader` has dual borders on hover.